pages/bellapps.md

```diff
@@ -1,4 +1,10 @@
 #BeLL-Apps: Communities and Nations
+
+##Objective
```

Comment:
Did you know, you need to add a `space` after the last hashtag `#`?
Because of you already on this page, please fix the header formatting as well.